.azure-pipelines-gh-pages.yml

```
@@ -19,7 +19,7 @@ jobs:
         env
         git status
         git rev-parse HEAD
-        git checkout -b master $BUILD_SOURCEVERSION
+        git checkout -b master $(git rev-parse HEAD)
```

```suggestion
        git checkout -b master HEAD
```

Seems to work just as well, but this is fine too.